src/reducers/__tests__/console.test.jsx

```diff
@@ -37,14 +38,19 @@ test('consoleValueProduced adds value to existing entry', () => {
 
 test('consoleErrorProduced adds error to existing entry', () => {
   const key = '123';
-  const name = 'NameError';
-  const message = 'bogus is not defined';
+  const error = {
+    name: 'TypeError',
+    raw: 'You tried to call `i()` as a function, but `i` is not a function.',
+    reason: 'not-a-function',
+    text: 'You tried to call `i()` as a function, but `i` is not a function.',
+    type: 'error',
+  };
```

Comment:
OK ONE MORE little thing here for total perfection: this kind of sentinel value for a piece of structured data is the perfect use case for a factory. Check out the factories in [`__factories__/clients`](https://github.com/popcodeorg/popcode/tree/master/__factories__/clients), for examples, although in this case I think the right place for your factory would be `__factories/validations/errors.js` since this object shape is based on the shape of the errors the validation module returns.

Reply:
Created `consoleErrorFactory` and added to `__factories/validations/errors.js`.